### PR TITLE
Clarify that fetch "credentials" value affects response credentials too

### LINKED ISSUE #2409: Issue with "WindowOrWorkerGlobalScope.fetch()": (short summary here please)
MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch

#### What information was incorrect, unhelpful, or incomplete?

The documentation for the "credentials" init parameter says that it affects whether credentials are set.

It also seems to control whether cookies in the response are accepted, at least in Chrome.  I personally observed this behavior and so did others: https://github.com/github/fetch/issues/386#issuecomment-243223273

#### Specific section or headline?

See above.

#### What did you expect to see?

MDN often has browser-specific notes.  This issue was pretty time-consuming to debug, so I think it would be worth calling it out in the MDN docs.

#### Did you test this? If so, how?

Using the network tab in Chrome.  See link to GitHub issue above.

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/api/windoworworkerglobalscope/fetch`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch
* GitHub URL: https://github.com/mdn/content/blob/master/files/en-us/web/api/windoworworkerglobalscope/fetch/index.html
* Last commit: https://github.com/mdn/content/commit/b39d9fe963b489691a0835cdb1bcc077168cd9c6
* Document last modified: 2021-02-11T16:28:01.000Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
This change updates the documentation on the `fetch()` method, to clarify that the `credentials` value affects whether browsers use credentials sent back in responses (e.g., any `Set-Cookie` response headers) — not just whether credentials are sent with the request. Fixes https://github.com/mdn/content/issues/2409. Related: https://github.com/whatwg/fetch/pull/1174.

@annevk, would be great if you could also review the 4843f89 part of this (the other commit is some automated lint fixing).

### CODE REVIEW

**Review by annevk — APPROVED** [review]
Looks good to me, thanks!

File: files/en-us/web/api/windoworworkerglobalscope/fetch/index.html [hunk 1]
```diff
@@ -57,7 +57,7 @@ <h3 id="Parameters">Parameters</h3>
         fetch. Some browsers accept the <code>blob:</code> and <code>data:</code> schemes.
       </li>
       <li>A {{domxref("Request")}} object.</li>
-      <li>A {{domxref("URL")}} object, or any other object with a <a href="https://developer.mozilla.org/en-US/docs/MDN/Contribute/Howto/Write_an_API_reference/Information_contained_in_a_WebIDL_file#stringifiers">stringifier</a> that provides the URL for the resource you want to fetch.</li>
+      <li>A {{domxref("URL")}} object, or any other object with a <a href="/en-US/docs/MDN/Contribute/Howto/Write_an_API_reference/Information_contained_in_a_WebIDL_file#stringifiers">stringifier</a> that provides the URL for the resource you want to fetch.</li>
```
[the remark this answers is not in the document]
Comment (Rumyra) [reply files/en-us/web/api/windoworworkerglobalscope/fetch/index.html:60]: Let me know if you want to resolve this @sideshowbarker - happy to approve either way honestly 👍

[the remark this answers is not in the document]
Comment (sideshowbarker) [reply files/en-us/web/api/windoworworkerglobalscope/fetch/index.html]: > > I would put this after the USVString entry as it's really a side-effect of it accepting that. "A string or any object with a stringifier, including URL objects, where the string or resulting string provides the URL ..."
> 
> Let me know if you want to resolve this @sideshowbarker - happy to approve either way honestly

Thanks yeah I agree with Anne’s point about this, so I went ahead and made the change (with slightly different wording)

[the remark this answers is not in the document]
Comment (Rumyra) [reply files/en-us/web/api/windoworworkerglobalscope/fetch/index.html]: 🙌

**Review by Rumyra — APPROVED** [review]

**Pull request merged**